tests/test_openms.py

```
@@ -1,6 +1,12 @@
 import pytest
 
-from sdrf_pipelines.openms.openms import get_openms_file_name, parse_tolerance
+from sdrf_pipelines.openms.openms import (
+    TMT_PLEXES,
+    OpenMS,
```

Ruff (0.8.2)

`sdrf_pipelines.openms.openms.OpenMS` imported but unused

Remove unused import: `sdrf_pipelines.openms.openms.OpenMS`

(F401)